src/attr.rs

```diff
@@ -1,68 +1,96 @@
+use crate::Result;
+
 use syn::{
-    parse::{Parse, ParseStream},
-    LitStr, Token,
+    parse::{Parse, ParseBuffer, ParseStream},
+    Attribute, LitStr, Token,
 };
 
-pub enum BotCommandAttribute {
+pub(crate) enum CommandAttrName {
     Prefix,
     Description,
-    RenameRule,
-    CustomParser,
+    Rename,
+    ParseWith,
     Separator,
 }
 
-impl Parse for BotCommandAttribute {
-    fn parse(input: ParseStream) -> Result<Self, syn::Error> {
+impl Parse for CommandAttrName {
+    fn parse(input: ParseStream) -> syn::Result<Self> {
         let name_arg: syn::Ident = input.parse()?;
+
         match name_arg.to_string().as_str() {
-            "prefix" => Ok(BotCommandAttribute::Prefix),
-            "description" => Ok(BotCommandAttribute::Description),
-            "rename" => Ok(BotCommandAttribute::RenameRule),
-            "parse_with" => Ok(BotCommandAttribute::CustomParser),
-            "separator" => Ok(BotCommandAttribute::Separator),
-            _ => Err(syn::Error::new(name_arg.span(), "unexpected argument")),
+            "prefix" => Ok(CommandAttrName::Prefix),
+            "description" => Ok(CommandAttrName::Description),
+            "rename" => Ok(CommandAttrName::Rename),
+            "parse_with" => Ok(CommandAttrName::ParseWith),
+            "separator" => Ok(CommandAttrName::Separator),
```

Comment:
Maybe it's better to add `use CommandAttrName::*;` prior to `match`?

Reply:
Makes sense 👍🏻 

Maybe we (you) should add this recommendation to style guide?

Reply:
Ok, will add 👍🏻